extensions/oidc-token-propagation-reactive/deployment/src/test/resources/application.properties

```diff
@@ -1,6 +1,8 @@
 quarkus.oidc.auth-server-url=${keycloak.url}/realms/quarkus
 quarkus.oidc.client-id=quarkus-app
 quarkus.oidc.credentials.secret=secret
+quarkus.oidc.application-type=hybrid
+quarkus.oidc.token.audience=any
```

Comment:
I had to add it because the wiremock sets an audience different to what this test expects for ID tokens by default (which `quarkus-app`), so I relaxed the audience verification for this test